### PR TITLE
feat: add view offices component

### PR DESCRIPTION
**Pull request by spider0061**
added the view offices component.

## Related issues and discussion
#635 

## Screenshots, if any
Screenshot from 2020-02-29 10-41-22
Screenshot from 2020-02-29 10-41-38

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/organization/offices/dataTable.resolver.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+@Injectable()
+export class DataTableResolver implements Resolve<Object> {
+
+  /**
+   * @param {OrganizationService} organizationService Organization service.
+   */
+  constructor(private organizationService: OrganizationService) {}
+
+  /**
+   * Returns the dataTable data.
```
Comment (luckyman20) [inline src/app/organization/offices/dataTable.resolver.ts]:
```suggestion
   * Returns the data table data.
```

File: src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    this.route.data.subscribe((data: { dataTable: any }) => {
+      this.dataTablesData = data.dataTable;
+    });
+    this.route.data.subscribe((data: { office: any }) => {
+      this.officeData = data.office;
+    });
+    this.route.data.subscribe((data: { tableDetails: any }) => {
+      this.tableDetailsData = data.tableDetails;
+    });
+    for (let i = 0; i < this.tableDetailsData.data.length; i++) {
```
Comment (luckyman20) [inline src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.ts]: Please do appropriate naming of the variables.

File: src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      this.tablesData.push(this.temp);
+
+    }
+  }
+
+  /**
+   * Sets the dataTable.
+   */
+  ngOnInit() {
+  this.setTableDetails();
```
Comment (luckyman20) [inline src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.ts]: formatting

File: src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
```
Comment (karantakalkar) [inline src/app/organization/offices/view-offices/notarios-tab/notarios-tab.component.html:1]: @spider0061 data tables are to be dynamically created like in clients view and not statically typed like this one.

**Comment by karantakalkar** [comment]
@spider0061 do you have time to implement data tables? if not please close this.

**Comment by spider0061** [comment]
@karantakalkar I will work on it once I get time. Closing for now.

**Pull request closed without merging**